doc/Learn/Maui/ThirdParty-DevExpress.md

```diff
@@ -26,22 +26,58 @@ Once you have an account with DevExpress, you need to visit the [Your DevExpress
 
 ## Getting Started
 
-1. Create a new application using the `unoapp` template, enabling .NET MAUI Embedding. In this case, we're going to use the Blank template (`-preset blank`) and include .NET MAUI Embedding support (`-maui`).
-
-    ```dotnetcli
-    dotnet new unoapp -preset blank -maui -o MauiEmbeddingApp
+### [Visual Studio](#tab/vs)
+ 
+> [!NOTE]
+> If you don't have the **Uno Platform Extension for Visual Studio** installed, follow [these instructions](xref:Uno.GetStarted.vs2022).
+ 
+- Launch **Visual Studio** and click on **Create new project** on the Start Window. Alternatively, if you're already in Visual Studio, click **New, Project** from the **File** menu.
+ 
+- Type `Uno Platform` in the search box
+ 
+- Click **Uno Platform App**, then **Next**
+ 
+- Name the project `DevExpressApp` and click **Create**
+ 
+At this point you'll enter the **Uno Platform Template Wizard**, giving you options to customize the generated application.
+ 
+- Select **Blank** in **Presets** selection
+ 
+- Select the **Platforms** tab and unselect **MacCatalyst** and **Windows** platforms
+ 
+For more information on all the template options, see [Using the Uno Platform Template](xref:Uno.GettingStarted.UsingWizard).
+ 
+- Click **Create** to complete the wizard
+ 
+The template will create a solution with a single cross-platform project, named `DevExpressApp`, ready to run.
+
+### [Command Line](#tab/cli)
+ 
+> [!NOTE]
+> If you don't have the Uno Platform dotnet new templates installed, follow [dotnet new templates for Uno Platform](xref:Uno.GetStarted.dotnet-new).
+ 
+Create a new application using the `unoapp` template, enabling .NET MAUI Embedding. In this case, we're going to use the Blank template (`-preset blank`) and include .NET MAUI Embedding support (`-maui`).
+ 
+    ```bash
+    dotnet new unoapp -preset blank -maui -platforms "android" -platforms "ios" -platforms "wasm" -platforms "desktop" -o DevExpressApp
```

Comment:
Does WASM and Desktop support MauiEmbedding?

Reply:
I have made the adjustment @eriklimakc